tests/unit/objectProvider.py

```diff
@@ -25,3 +26,75 @@ def __init__(self, role=controlTypes.ROLE_UNKNOWN,**kwargs):
 	def _get_name(self):
 		return controlTypes.roleLabels.get(self.role,controlTypes.ROLE_UNKNOWN)
 
+class NVDAObjectWithDecoratedScript(PlaceholderNVDAObject):

```

Comment:
I intended this module as a general utility module; i.e. code that multiple tests might use. I imagine multiple tests might want a placeholder NVDAObject. In contrast, these classes seem rather test specific: more like test data than utilities. I'd be inclined to move them into the module with the tests.